config.json

```diff
@@ -244,6 +244,18 @@
         "Result"
       ]
     },
+    {
+      "uuid": "10923b0b-c726-44a7-9e02-b775e7b8b237",
+      "slug": "simple-linked-list",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 4,
+      "topics": [
+        "lists",
+        "type_conversion",
+        "struct"
+        ]
```

Comment:
this should be indented two spaces less than it currently is, so that it's on the same column as "topics". the other exercises follow this pattern.